src/server/utilities/cspHeader/index.js

```diff
@@ -299,6 +303,7 @@ export const generateDefaultSrc = () => ["'self'"];
 export const generateFontSrc = () => [
   'https://gel.files.bbci.co.uk',
   'https://ws-downloads.files.bbci.co.uk',
+  'https://static.bbci.co.uk', // STY includes
```

Comment:
The `generateFontSrc` function is currently platform unaware (i.e. doesn't accept `isAmp` or `isLive`) , so this directive will be added to the header of _all_ responses (including on AMP). Is this intentional?

Reply:
not really intentional per se, but just following the convention, a better option would be transforming to add `https://static.bbci.co.uk` depending on the platform but I wanted to know what others think of it/doing that in this pr

Reply:
If it's not intentional, we should update this function, so that the directive is only applied to the required platforms. This PR https://github.com/bbc/simorgh/pull/6209/files has an example of how `generateImgSrc` was updated to do that.

Reply:
I'd agree with the change @simonsinclair suggests, given includes will load in an iframe on amp, we definitely do not need this directive on amp responses. Additionally, now is good time to make changes like this when we have less services live imo.